fog/test-client/src/test_client.rs

```diff
@@ -144,6 +146,7 @@ impl TestClient {
             policy,
             grpc_retry_config,
             account_keys,
+            network_id,
```

Comment:
Would it make sense to check here that test client isn't ever sending to prod or something like that, just to avoid that scenario?

Reply:
no, test-client IS used in production as a canary that ensures that consensus and fog are working by continuous end to end tests.

Reply:
Oh. Thanks. I thought that was a separate test.